SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/EdgeSetGeometryAlgorithms.inl

```diff
@@ -560,19 +560,19 @@ sofa::type::vector< double > EdgeSetGeometryAlgorithms<DataTypes>::computePointP
 
     // Compute Coord of second edge XH:
 
-    sofa::type::Vec<3,double> AB; DataTypes::get(AB[0], AB[1], AB[2], coord_AB);
-    sofa::type::Vec<3,double> AC; DataTypes::get(AC[0], AC[1], AC[2], coord_AC);
-    sofa::type::Vec<3,double> ortho_ABC = cross (AB, AC)*1000;
-    sofa::type::Vec<3,double> coef_CH = cross (ortho_ABC, AB)*1000;
+    sofa::type::Vec<3,SReal> AB; DataTypes::get(AB[0], AB[1], AB[2], coord_AB);
+    sofa::type::Vec<3,SReal> AC; DataTypes::get(AC[0], AC[1], AC[2], coord_AC);
+    sofa::type::Vec<3,SReal> ortho_ABC = cross (AB, AC)*1000;
+    sofa::type::Vec<3,SReal> coef_CH = cross (ortho_ABC, AB)*1000;
```

Comment:
Again, here I don't see why it's a `SReal`. I suspect that this practice is very common in topologies, and I would go with `DataTypes::Real`

Reply:
for this comment (and partially the one before) I agree completely